dp-sample.yml

```diff
@@ -3,26 +3,37 @@ AWSTemplateFormatVersion: '2010-09-09'
 Description: Data provider for AWS services
 
 Parameters:
+  HostedZoneId:
+    Type: AWS::Route53::HostedZone::Id
+    Description: Route53 Hosted Zone ID
   Version:
     Type: Number
     Description: "Just for testing"
     Default: 1
 
-
 Resources:
-  # VpcInfo - VPC Lookup
+  ZoneInfo:
+    Type: Custom::R53HostedZone
+    Properties:
+      ServiceToken:
+        Fn::ImportValue: !Sub "cfn:cfn-dataprovider:${AWS::Region}:arn"
+      HostedZoneId: !Ref HostedZoneId
+      Tags:
+        - Key: Version
+          Value: !Ref Version
+
   SubnetInfo:
     Type: Custom::Subnet
     Properties:
       ServiceToken:
         Fn::ImportValue: !Sub "cfn:cfn-dataprovider:${AWS::Region}:arn"
       Filter:  # For more options check https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_subnets
-        cidr: 172.31.32.0/20
-        default-for-az: True
+        #cidr: 172.31.32.0/20
```

Comment:
Instead of edit out, might worth using the same pattern like for the Route53 ( aka use the Parameters )